radicle-node/src/worker.rs

```diff
@@ -403,6 +291,155 @@ impl<G: Signer + Ecdh + 'static> Worker<G> {
             }
         }
     }
+
+    fn _upload_pack(
+        &self,
+        fetch: &Fetch,
+        pktline_r: &mut pktline::Reader<WireReader>,
+        stream_w: &mut WireWriter<G>,
+    ) -> Result<Vec<RefUpdate>, FetchError> {
```

Comment:
What do methods with preceding '_' signify?  I notice process(..) calls _process(..) but _upload_pack(..) calls upload_pack(..).

Reply:
Thanks for catching this. I've updated the code by swapping them around :)